### ISSUE TITLE
minSpareRows and minSpareCols breaks Handsontable rendering

### ISSUE THREAD

**Issue by lunfel** [handsontable/handsontable#5917]
### Description

The minSpareRows and minSpareCols do not behave as it is supposed to. Handsontable will not render data past the values of minSpareRows and minSpareCols.

### Steps to reproduce

1. Setup a Vue Handsontable with data: [[]]
2. put minSpareRows: 6
3. put minSpareCols: 4
4. Then after component is loaded, setTimeout (imitate ajax behavior) to load data by doing hotSettings.data = myNewData
5. Handsontable will not render more than the 6th row and no more than 4 columns event if loaded data have more rows and columns.

### Demo

https://jsfiddle.net/x3gaqndu/3/

### Your environment
* Vue wrapper version: 4.0.0
* Handsontable version: 7.0.0
* Browser Name and version: Chrome 73.0.3683.86
* Operating System: Ubuntu 18.04

**Comment by AMBudnik** [comment]
But.. I can see that this issue has been already raised here https://github.com/handsontable/handsontable/issues/4576 so I'm closing this topic.

**Comment by AMBudnik** [comment]
HI @lunfel 

We just fixed an issue related to yours. Please update to 8.0.0-beta1 and check if it solved for you as well.

Below are the resources

```html
</style><script src="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.js"></script><link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.css">
```

And this is how you can download the beta from NPM

`npm i handsontable@beta`

**Comment by AMBudnik** [comment]
hi @lunfel 

the referenced issue https://github.com/handsontable/handsontable/issues/4576 has just been closed as solved. This issue should be fixed after updating to the stable version 8.0.0
